### ISSUE TITLE
Multiple Job Types in processJobs

### ISSUE THREAD

**Issue by danielparas** [vsivsi/meteor-job#13]
Hi! I just updated to the latest release and realised that an array of job types can no longer be passed as a parameter for processJobs. Eg:

```
workerInstance = mJob.processJobs('job_queue', ['jobType1','jobType2'], workerOptions, function(job, callback){});

```

This invokes the error: `JobQueue: Invalid type, must be nonempty string`

Is this an intentional fix or a bug?

Thanks
Daniel

**Comment by vsivsi** [comment]
Sounds like a bug, I'll look into it today.

**Comment by vsivsi** [comment]
Confirmed bug:  https://github.com/vsivsi/meteor-job/commit/6307a1e6f5b40aa5fd9233b3a10fc994f38116dd#commitcomment-21882886

I'll fix today.

**Comment by danielparas** [comment]
Thanks for the quick reply! If I can help with anything let me know.

**Comment by vsivsi** [comment]
Fixed in version 1.5.1 of both npm and Atmosphere packages.
